content/authorization/guides/system-vendor/consent/liststatuschanges/_index.en.md

```
@@ -0,0 +1,75 @@
+---
+title: List latest status changes for consent
+linktitle: List latest status changes for consent
+weight: 10
+toc: false
+---
+
+### Prerequisites
```

markdownlint-cli2 (0.22.1)

[warning] Heading levels should only increment by one level at a time
Expected: h2; Actual: h3

(MD001, heading-increment)